pypose/module/imu_preintegrator.py

```diff
@@ -12,30 +12,43 @@ class IMUPreintegrator(nn.Module):
         rot (pypose.SO3, optional): initial rotation. Default: :meth:`pypose.identity_SO3`
         vel (torch.Tensor, optional): initial position. Default: torch.zeros(3)
         gravity (float, optional): the gravity acceleration. Default: 9.81007
-        gyro_cov (float, optional): covariance of the gyroscope. Default: (1.6968e-4)**2
-        acc_cov (float, optional): covariance of the accelerator. Default: (2e-3)**2
+        gyro_cov: covariance of the gyroscope. If the covariance of the x, y and z axises are
+            different, the :obj:`gyro_cov` use the form :code:`torch.Tensor(gyro_cov_x, gyro_cov_y,
+            gyro_cov_z)`. Default: (3.2e-3)**2
+        acc_cov: covariance of the accelerator. If the covariance of the x, y and z axises are
+            different, the :obj:`acc_cov` use the form :code:`torch.Tensor(acc_cov_x, acc_cov_y,
+            acc_cov_z)`. Default: (8e-2)**2
```

Comment:
Suggest to change: covariance of the gyroscope. Default: (3.2e-3)**2. Please use the form of :code:`torch.Tensor(gyro_cov_x, gyro_cov_y, gyro_cov_z)`, if the covariance on the three axes is different.

Reply:
That's a good suggestion, I just updated.